sim/common/sod/item_effects/phase_5.go

```diff
@@ -161,25 +185,122 @@ func init() {
 		})
 	})
 
-	core.AddEffectsToTest = true
-}
+	// https://www.wowhead.com/classic/item=231784/lightnings-cell
+	//
 
-	// https://www.wowhead.com/classic/spell=465414/crusaders-zeal
-	// Used by:
-	// - https://www.wowhead.com/classic/item=229806/truthbearer and
-	// - https://www.wowhead.com/classic/item=229749/truthbearer
-	func crusadersZealAura465414(character *core.Character) *core.Aura {
-		return character.GetOrRegisterAura(core.Aura{
-			ActionID: core.ActionID{SpellID: 465414},
-			Label:    "Crusaders Zeal (465414)",
-			Duration: time.Second * 8,
+	// Use: Energizes a Paladin with light, increasing melee attack speed by 25%
+	// and spell casting speed by 33% for 20 sec. (2 Min Cooldown)
+	core.NewItemEffect(ScrollsOfBlindingLight, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		duration := time.Second * 20
+
+		aura := character.GetOrRegisterAura(core.Aura{
+			Label:    "Blinding Light",
+			ActionID: core.ActionID{SpellID: 467522},
+			Duration: duration,
 			OnGain: func(aura *core.Aura, sim *core.Simulation) {
-				character.PseudoStats.BonusDamage += 15
-				character.MultiplyAttackSpeed(sim, 1.30)
+				character.MultiplyAttackSpeed(sim, 1.25)
+				character.MultiplyCastSpeed(1.33)
 			},
 			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
-				character.PseudoStats.BonusDamage -= 15
-				character.MultiplyAttackSpeed(sim, 1/1.30)
+				character.MultiplyAttackSpeed(sim, 1.0/1.25)
+				character.MultiplyCastSpeed(1.0 / 1.33)
+			},
+		})
+
+		spell := character.GetOrRegisterSpell(core.SpellConfig{
+			ActionID: core.ActionID{ItemID: ScrollsOfBlindingLight},
+			Flags:    core.SpellFlagNoOnCastComplete,
+
+			Cast: core.CastConfig{
+				CD: core.Cooldown{
+					Timer:    character.NewTimer(),
+					Duration: time.Minute * 2,
+				},
+				SharedCD: core.Cooldown{
+					Timer:    character.GetOffensiveTrinketCD(),
+					Duration: duration,
+				},
+			},
+
+			ApplyEffects: func(sim *core.Simulation, _ *core.Unit, _ *core.Spell) {
+				aura.Activate(sim)
 			},
 		})
-	}
\ No newline at end of file
+
+		character.AddMajorCooldown(core.MajorCooldown{
+			Type:     core.CooldownTypeDPS,
+			Priority: core.CooldownPriorityDefault,
+			Spell:    spell,
+		})
+	})
+
+	core.NewSimpleStatOffensiveTrinketEffect(WrathOfWray, stats.Stats{stats.Strength: 92}, time.Second*20, time.Minute*2)
+
+	core.AddEffectsToTest = true
+}
+
+func blazefuryTriggerAura(character *core.Character, spellID int32, spellSchool core.SpellSchool, damage int32) *core.Aura {
+
+	procSpell := character.GetOrRegisterSpell(core.SpellConfig{
+		ActionID:         core.ActionID{SpellID: spellID},
+		SpellSchool:      core.SpellSchoolHoly,
```

Comment:
Should this use the `spellSchool` param?

Reply:
yeah